src/s3b_the_roach_market.tw2

```diff
@@ -98,13 +93,43 @@ I unrolled the top and peeked inside. “Wow. This is, uh, a pretty full bag.”
 
   Dan tapped the side of his nose with his finger. “Trip in peace, my ‘lette. Dopeman ain’t never snitched on nobody.”
 
-  I nodded, and made my way deeper into the Roach Market.
-
-  > [[Head to Auntie’s gift shop|RM-giftshop-entrance]]
-  > [[Head to the junk heap|RM-junkheap-entrance]]
+  I nodded, and made my way [[deeper|RM-drugs-second-thoughts]] into the Roach Market.
   ]
 ]
 
+::RM-drugs-second-thoughts
+I rounded a corner as I headed away from Dan (who was already starting to nod off again in his camp chair) and passed by a metal barrel overflowing with garbage.
+
+There were dozens just like this one all over Tent City: emptied out weekly by order of the Honchos once upon a time, now left to sit until some ninja got fed up and dumped ‘em out someplace — usually straight into the river that bent around Gatherville. Juggalo altruism had its limits.
+
+(click: "limits")[$dissolve[Most of what spilled onto the ground from this barrel looked to be paper bags discarded after other ninjas’ trips to The Stash.
+
+My backpack was still hung unzipped from my hand, my own paper bag sitting at the top where I stuffed it just moments ago. Free drugs is free drugs, yeah, but now I wasn't so sure if I wanted to tote this around all night like a schoolkid handed a bag lunch.
```

Comment:
"was still hung" => "hung" maybe? And the paper bag was Dan's until just recently, so I might just say:

"My backpack hung unzipped from my hand, the paper bag sitting at the top where I stuffed it just moments ago."

Reply:
Yeah I'd trim the former, leave the latter (so as to keep the connection between mass of anonymous discarded paper bags mentioned before, vs. my own paper bag, perhaps to join them soon?)

Reply:
Ahhh, got it. I like that. Sounds good! I'll change "was still hung" to "hung" and leave the rest.